libs/community/tests/integration_tests/retrievers/test_contextual_compression.py

```diff
@@ -24,3 +25,25 @@ def test_contextual_compression_retriever_get_relevant_docs() -> None:
     actual = retriever.invoke("Tell me about the Celtics")
     assert len(actual) == 2
     assert texts[-1] not in [d.page_content for d in actual]
+
+
+@pytest.mark.asyncio
```

Comment:
```suggestion
```

Reply:
if I delete this annotation, the test is not run.